### DISCUSSION TITLE
What's the equivalent of @SqsListener(value = "myQueue", deletionPolicy = SqsMessageDeletionPolicy.NO_REDRIVE) in 3.1.0 ?

### DISCUSSION THREAD

**Discussion by sandronm** [awspring/spring-cloud-aws#1059] [Q&A]
Hello all!

I'm starting a migration from spring-cloud-aws 2.4.2 to the latest 3.1.0 (so also from AWS SDK 1 to 2) and I'm wondering what's the equivalent of the `SqsMessageDeletionPolicy.NO_REDRIVE` now?

I know, thank to that issue https://github.com/awspring/spring-cloud-aws/issues/761 that `delationPolicy` has been replaced by `acknownledgementMode` but unfortunately the value `NO_REDRIVE` doesn't exist.

Could help me? I could help with the migration guide somehow https://docs.awspring.io/spring-cloud-aws/docs/3.1.0/reference/html/index.html#migration-from-2-x-to-3-x

**Comment by tomazfernandes** [comment]
> SqsMessageDeletionPolicy.NO_REDRIVE

We don't currently have an equivalent policy. In all honesty, I don't really understand what it's used for - seems like a weird concept to me.

Can you please share what exactly is your use case and what do you need it for?

Maybe @maciejwalkowiak and @MatejNedic can provide a bit more context on why was this policy added?

Thanks.
